source/leuven/slurm_specifics.rst

```diff
@@ -0,0 +1,172 @@
+.. _leuven_slurm_specifics:
+
+Site-specific Slurm info
+========================
+
+While the :ref:`Running jobs in Slurm <running jobs>` pages provide generic
+information regarding Slurm, there are additional points to consider when
+when using Slurm on Tier-2 clusters hosted at KU Leuven.
+
+
+Compute credits
+---------------
+When submitting a job, you need to provide a valid Slurm credit account holding
+enough compute credits for the job using the ``-A/--account`` option.
+For more information, please consult the following pages:
+
+.. toctree::
+   :maxdepth: 1
+
+   ./credits
+   ./slurm_accounting
+
+
+.. _leuven_job_shell:
+
+Job shell
+---------
+For batch jobs we strongly recommend to use ``#!/bin/bash -l`` as the shebang
+at the top of your jobscript. The ``-l`` option is needed to make sure that
+your ``~/.bashrc`` settings get applied and the appropriate :ref:`cluster
+module <cluster_module>` gets loaded at the start of the job.
+This is not strictly needed for interactive jobs: ``srun ... --pty bash``
+and ``srun ... --pty bash -l`` will give essentially identical environments.
+
+Note that we still discourage loading modules in your ``~/.bashrc`` file and
+recommend to do that in your jobscripts instead (see also the
+:ref:`Compiling software for wICE<wice_compilation>` paragraph for example).
+
+
+.. _leuven_cluster_choice:
+
+Cluster choice
+--------------
+Many Slurm commands (like `sbatch`, `srun`, `scontrol`, `squeue`, `scancel`,
+...) accept a ``-M/--clusters`` option which selects one or more clusters.
+The default value depends on where the command is executed (Genius for the
+Genius compute nodes and login nodes; wICE for the wICE compute nodes).
+This means that if you are connected to a (Genius) login node, you will need
+to add ``-M wice`` in order to select wICE instead of Genius. In order to
+avoid potential mistakes we have made the ``-M/--clusters`` option mandatory
+when submitting jobs.
+
+
+.. _leuven_job_monitoring:
+
+Monitoring jobs
+---------------
+
+For monitoring or debugging jobs, you can look into the following Slurm tools:
+
+* `scontrol <https://slurm.schedmd.com/scontrol.html>`__ to view Slurm
+  configuration and state
+* `squeue <https://slurm.schedmd.com/squeue.html>`__ to get information about
+  jobs in the scheduling queue
+* `sacct <https://slurm.schedmd.com/sacct.html>`__ to display information about
+  finished jobs
+
+.. note::
+
+    Don't forget the ``-M/--clusters`` option for these commands, as mentioned
```

Comment:
Here it might be helpful to mention that the `--clusters` option accepts `ALL`. For instance if you want to have an overview of jobs on all clusters, run `squeue -M ALL`.

Reply:
(added in 3c7c627bd661f5bab97f6ffe79d3099ede034619)